pipeline/07-export.R

```diff
@@ -174,13 +175,50 @@ assessment_pin10_prepped <- assessment_pin_prepped %>%
 for (town in unique(assessment_pin_prepped$township_code)) {
   message("Now processing: ", town_convert(town))
 
+  workbook_name <- glue(
+    params$assessment$year,
+    str_replace(town_convert(town), " ", "_"),
+    "Initial_Model_Values_Condo.xlsx",
+    .sep = "_"
+  )
```

Comment:
The workbook name is now used for linking to the `Buildings` sheet, so we need to define it earlier in the stage.